docs/source/index.md

```diff
@@ -24,7 +24,7 @@ Lacth SDK allows developers to upload workflows to the full-featured [Latch Plat
 
 With Latch SDK, developers can write the description to their workflow and customize input parameters using plain Markdown. Latch automatically parses the written text and Python function headers to compile a type-safe UI.
 
-**Specifying arbitrary cloud compute and storage resources for bioinformatics pipelines is difficult.** With Latch SDK, there are several Python task decorators that easily allow you to define the resources available at runtime. The framework starts at 2 CPUs and 4 GBs of memory and goes all the way to 31 CPUs, 120 GBs of memory and 1 GPU (24 GBs of VRAM, 9,216 CUDA cores) to easily handle all processing needs.
+**Specifying arbitrary cloud compute and storage resources for bioinformatics pipelines is difficult.** With the Latch SDK, there are several Python task decorators that easily allow you to define the resources available at runtime. The framework starts at 2 CPUs and 4 GiBs of memory and goes all the way to 95 CPUs, 490 GBs of memory, 4949 GiB of storage, and 1 GPU (24 GBs of VRAM, 9,216 CUDA cores) to easily handle all processing needs.
```

Comment:
note that it's not all at the same time (i.e. CPU limit for big-ram nodes is lower)

"GB" -> "GiB"